### DISCUSSION TITLE
Korg Triton Pro X Use Bluescsi

### DISCUSSION THREAD

**Discussion by aihk** [erichelgeson/BlueSCSI-v1#238] [Q&A]
I used it in Korg Triton Pro X
Bluescsi is formatting in Korg Triton Pro X that can be stored normally. But when I want to copy the files from the simulator, MAC's simulation prompts formatting。Is there any way to copy the file inside?
.hda file sent to your email

**Comment by erichelgeson** [comment]
You formatted in a Korg, then tried to use a Macintosh emulator to copy files to the formatted image? The instructions for copying files to a Macintosh image are specific to Macintosh and can't be used with other devices. What file system does Korg use? You'll have to find a utility that allows you to interact with that format.

**Comment by aihk** [comment]
I just do n’t know what program can interact, BlueScsi can be used on Korg. I want to ask you, do you know what program can interact? Or can it be like this video https://www.youtube.com/watch?v=2FSK2AUXOCK
